zendriver/core/browser.py

```diff
@@ -117,15 +118,17 @@ def __init__(self, config: Config, **kwargs):
                 )
             )
         # weakref.finalize(self, self._quit, self)
-        self.config = config
+
+        # each instance gets it's own copy so this class gets a copy that it can
+        # use to help manage the browser instance data (needed for multiple browsers)
+        self.config = copy.deepcopy(config)
 
         self.targets: List = []
         """current targets (all types)"""
         self.info: ContraDict | None = None
         self._target = None
         self._process = None
         self._process_pid = None
-        self._keep_user_data_dir = None
```

Comment:
Looks like this is cargo cult code. Unused by anything and I can see how this was handled a little differently in the final code.